test/util.mjs

```diff
@@ -11,7 +11,7 @@ export const {
   },
 } = globalThis;
 
-export const isSubsetJSON = (json1, json2, path) => {
+export const matchJSON = (json1, json2, path) => {
```

Comment:
It's still matching objects, not JSON, right? ;)

Reply:
Nah, we are only matching against json data structures. Just to be clear, I used `json` in my code as a type for json datastructures and not json-formatted string. The rest of the code base also follows this convention.